src/ray/common/ray_config_def.h

```diff
@@ -20,8 +20,8 @@ RAY_CONFIG(int64_t, ray_cookie, 0x5241590000000000)
 /// warning is logged that the handler is taking too long.
 RAY_CONFIG(int64_t, handler_warning_timeout_ms, 100)
 
-/// The duration between heartbeats. These are sent by the raylet.
-RAY_CONFIG(int64_t, heartbeat_timeout_milliseconds, 100)
+/// The duration between heartbeats send from the raylets.
```

Comment:
```suggestion
/// The duration between heartbeats sent to other raylets.
```

Reply:
This would actually be a bit misleading, because the heartbeats are sent from the raylets to the GCS, not directly to other raylets. Will fix wording though.